### ISSUE TITLE
--get-url option not work in aws ec2 environment from a week ago

### ISSUE THREAD

**Issue by kyoyagami** [rg3/youtube-dl#8241]
I run two linux environments. then, One is on VMware and another one is on AWS.

1. VMware
youtube-dl -v https://www.youtube.com/watch?v=FrG4TEcSuRg
[debug] System config: []
[debug] User config: []
[debug] Command-line args: [u'-v', u'https://www.youtube.com/watch?v=FrG4TEcSuRg']
[debug] Encodings: locale UTF-8, fs UTF-8, out UTF-8, pref UTF-8
[debug] youtube-dl version 2016.01.09
[debug] Python version 2.7.5 - Linux-3.10.0-123.el7.x86_64-x86_64-with-centos-7.0.1406-Core
[debug] exe versions: none
[debug] Proxy map: {}
[youtube] FrG4TEcSuRg: Downloading webpage
[youtube] FrG4TEcSuRg: Downloading video info webpage
[youtube] FrG4TEcSuRg: Extracting video information
[youtube] {22} signature length 41.43, html5 player en_US-vflGR-A-c
[youtube] {43} signature length 41.43, html5 player en_US-vflGR-A-c
[youtube] {18} signature length 41.43, html5 player en_US-vflGR-A-c
[youtube] {5} signature length 41.43, html5 player en_US-vflGR-A-c
[youtube] {36} signature length 41.43, html5 player en_US-vflGR-A-c
[youtube] {17} signature length 41.43, html5 player en_US-vflGR-A-c
[youtube] {137} signature length 41.43, html5 player en_US-vflGR-A-c
[youtube] {136} signature length 41.43, html5 player en_US-vflGR-A-c
[youtube] {135} signature length 41.43, html5 player en_US-vflGR-A-c
[youtube] {134} signature length 41.43, html5 player en_US-vflGR-A-c
[youtube] {133} signature length 41.43, html5 player en_US-vflGR-A-c
[youtube] {160} signature length 41.43, html5 player en_US-vflGR-A-c
[youtube] {140} signature length 41.43, html5 player en_US-vflGR-A-c
[youtube] {171} signature length 41.43, html5 player en_US-vflGR-A-c
[youtube] {249} signature length 41.43, html5 player en_US-vflGR-A-c
[youtube] {250} signature length 41.43, html5 player en_US-vflGR-A-c
[youtube] {251} signature length 41.43, html5 player en_US-vflGR-A-c
[youtube] FrG4TEcSuRg: Downloading DASH manifest
[debug] Invoking downloader on u'https://r1---sn-n3cgv5qc5oq-bh2k.googlevideo.com/videoplayback?mime=video%2Fmp4&key=yt6&upn=QNh6JFv2bFw&expire=1452865075&initcwndbps=3131250&sver=3&ratebypass=yes&lmt=1449035687131232&fexp=9407156%2C9408354%2C9413137%2C9416126%2C9418203%2C9419451%2C9420452%2C9422541%2C9422596%2C9423347%2C9423642%2C9423662%2C9424337%2C9425538%2C9425701%2C9427107&dur=245.063&source=youtube&mn=sn-n3cgv5qc5oq-bh2k&requiressl=yes&sparams=dur%2Cgcr%2Cid%2Cinitcwndbps%2Cip%2Cipbits%2Citag%2Clmt%2Cmime%2Cmm%2Cmn%2Cms%2Cmv%2Cpcm2%2Cpl%2Cratebypass%2Crequiressl%2Csource%2Cupn%2Cexpire&gcr=kr&ip=175.113.135.22&mm=31&itag=22&pcm2=no&ipbits=0&pl=13&id=o-AMszfIN8169ZdvGZYlJy6ncarTlwvkIhkQ0TcGkTaLW5&mt=1452843401&mv=m&ms=au&signature=32D957E9E792882D600243711A2478BD226BF340.290F2A87B5A31C320E3E62A3BB4BB64D09A2E361'
[download] Destination: PSY - DADDY(feat. CL of 2NE1) M_V-FrG4TEcSuRg.mp4
[download] 100% of 74.17MiB in 00:12

2. AWS
youtube-dl -v https://www.youtube.com/watch?v=FrG4TEcSuRg
[debug] System config: []
[debug] User config: []
[debug] Command-line args: [u'-v', u'https://www.youtube.com/watch?v=FrG4TEcSuRg']
[debug] Encodings: locale UTF-8, fs UTF-8, out UTF-8, pref UTF-8
[debug] youtube-dl version 2016.01.14
[debug] Python version 2.7.5 - Linux-3.10.0-123.8.1.el7.x86_64-x86_64-with-centos-7.0.1406-Core
[debug] exe versions: none
[debug] Proxy map: {}
[youtube] FrG4TEcSuRg: Downloading webpage
[youtube] FrG4TEcSuRg: Downloading video info webpage
[youtube] FrG4TEcSuRg: Extracting video information
[youtube] {22} signature length 41.43, html5 player en_US-vflGR-A-c
[youtube] {43} signature length 41.43, html5 player en_US-vflGR-A-c
[youtube] {18} signature length 41.43, html5 player en_US-vflGR-A-c
[youtube] {5} signature length 41.43, html5 player en_US-vflGR-A-c
[youtube] {36} signature length 41.43, html5 player en_US-vflGR-A-c
[youtube] {17} signature length 41.43, html5 player en_US-vflGR-A-c
[youtube] {137} signature length 41.43, html5 player en_US-vflGR-A-c
[youtube] {136} signature length 41.43, html5 player en_US-vflGR-A-c
[youtube] {135} signature length 41.43, html5 player en_US-vflGR-A-c
[youtube] {134} signature length 41.43, html5 player en_US-vflGR-A-c
[youtube] {133} signature length 41.43, html5 player en_US-vflGR-A-c
[youtube] {160} signature length 41.43, html5 player en_US-vflGR-A-c
[youtube] {140} signature length 41.43, html5 player en_US-vflGR-A-c
[youtube] {171} signature length 41.43, html5 player en_US-vflGR-A-c
[youtube] {249} signature length 41.43, html5 player en_US-vflGR-A-c
[youtube] {250} signature length 41.43, html5 player en_US-vflGR-A-c
[youtube] {251} signature length 41.43, html5 player en_US-vflGR-A-c
[youtube] FrG4TEcSuRg: Downloading DASH manifest
[debug] Invoking downloader on u'https://r5---sn-oguesnzl.googlevideo.com/videoplayback?ms=au&mt=1452842785&gcr=jp&sparams=dur%2Cgcr%2Cid%2Cip%2Cipbits%2Citag%2Clmt%2Cmime%2Cmm%2Cmn%2Cms%2Cmv%2Cnh%2Cpl%2Cratebypass%2Crequiressl%2Csource%2Cupn%2Cexpire&ip=52.69.147.21&mv=u&id=o-ACNFJNn3iRJzDLyeMGZ38nm0kzXM4w0PGbM5Lg42oHQM&mm=31&mn=sn-oguesnzl&mime=video%2Fmp4&key=yt6&nh=IgpwcjAyLm5ydDE5KgkxMjcuMC4wLjE&requiressl=yes&ratebypass=yes&fexp=9416126%2C9420452%2C9422596%2C9423662&lmt=1449035687131232&expire=1452865154&sver=3&ipbits=0&dur=245.063&source=youtube&upn=5Otr_u6pt6I&itag=22&pl=24&signature=246100E2FE4CE0D4CA63601EDE4FE12397BDD465.336A289F6F4EFBC0EABEC38EBE4BE32651C9275E'
[download] Destination: PSY - DADDY(feat. CL of 2NE1) M_V-FrG4TEcSuRg.mp4
[download] 100% of 74.17MiB in 00:00

Basically, everything looks fine. Even downloading the video completed successfully both.
But i can only see video in internet browser with url that generated on my VMware environment.
I mean that video url is not valid only on my AWS environment while downloading video works fine.
It had been worked till a week ago. but now it hasn't been working.
Anyone have any idea for this issue?
Thanks
Regards

**Comment by yan12125** [comment]
See [this FAQ entry](https://github.com/rg3/youtube-dl#i-extracted-a-video-url-with--g-but-it-does-not-play-on-another-machine--in-my-webbrowser). In short: URLs extracted from ```-g``` is not guaranteed to be usable on other machines.
